common/taskana-common-data/src/main/resources/sql/sample-data/task.sql

```diff
@@ -1,96 +1,96 @@
+-- TASK TABLE          (ID                                        , EXTERNAL_ID                               , CREATED             , CLAIMED              , COMPLETED            , modified             , received             , planned              , due                  , name                      , creator                      , description                                , note                                     , priority, manual_priority, state       , classification_category  , classification_key, classification_id                         , workbasket_id                              , workbasket_key, domain    , business_process_id, parent_business_process_id, owner         , por_company , por_system , por_system_instance, por_type , por_value    , is_read, is_transferred, callback_info, callback_state, custom_attributes ,custom1      ,custom2,     ,custom3,    ,custom4    ,custom5      ,custom6     ,custom7     ,custom8     ,custom9     ,custom10    ,custom11,    ,custom12     ,custom13     ,custom14  ,custom15     ,custom16    , custom-int-1, custom-int-2, custom-int-3, custom-int-4, custom-int-5, custom-int-6, custom-int-7, custom-int-8
+INSERT INTO TASK VALUES('TKI:000000000000000000000000000000000000', 'ETI:000000000000000000000000000000000000', RELATIVE_DATE(-1)   , RELATIVE_DATE(0)     , null                 , RELATIVE_DATE(0)     , RELATIVE_DATE(0)     , RELATIVE_DATE(-1)    , RELATIVE_DATE(0)     , 'Task99'                  , 'creator_user_id'            , 'Lorem ipsum was n Quatsch dolor sit amet.', 'Some custom Note'                       , 1       , -1             , 'CLAIMED'   , 'MANUAL'                 , 'T2000'           , 'CLI:100000000000000000000000000000000016', 'WBI:100000000000000000000000000000000006' , 'USER-1-1'    , 'DOMAIN_A', 'BPI21'            , 'PBPI21'                  , 'user-1-1'  , 'MyCompany1', 'MySystem1', 'MyInstance1'      , 'MyType1', 'MyValue1'     , true   , false         , null         , 'NONE'        , null              , 'custom1'   , 'custom2'   , 'custom3'  , 'custom4' , 'custom5'   , 'custom6'  , 'custom7'  , 'custom8'  , 'custom9'  , 'custom10' , 'custom11'  , 'custom12'  , 'custom13'  , 'abc'    , 'custom15'  , 'custom16' , 1           , 2           , 3           , 4           , 5           , 6           , 7           , 8   );
```

Comment:
For me TASKANA is not booting like this. After executing the udate script, the columns get added after custom 16. The first new column is the timestamp "received". But here in this script an additional timestamp is put after planned. This won't work like this.

Reply:
This doesn't matter for the test data.
The test data assumes that no update script has been applied

Reply:
Since you've already test data in the database, please disable testdata generation for the second boot (with updated tables).